### ISSUE TITLE
will this project be updated with latest code or there is some similar project like this?

### ISSUE THREAD

**Issue by hileon** [darwin/cmx.js#8]
I cloned the code and try to install in local, but it seems the packages it depends on are mostly out of dated. 
Not sure if there is someone there have already fork and update the project, or is there some similar projects like cmx.js that is built recently?

**Comment by Promaethius** [comment]
@hileon you can get the site working by bypassing the old grunt dependency hell and using an older version of Node that still conforms to the tis changes. I'm using https://nodejs.org/en/blog/release/v0.12.18
```
nom install --save-dev coffeescript@~1.4.0
./node_modules/.bin/coffee -c **/*.coffee
cd app && python3 -m http.server 8080
```
It'll get you the trans-pilled comic editor and you can host it wherever.

**Comment by James471** [comment]
@Promaethius Thanks for this, however, the visual editor isn't working. The visual editor area is just blank. Do you have an idea?
![image](https://github.com/darwin/cmx.js/assets/56861963/6f61b83d-9123-4a49-b7ac-776962b543cb)

**Comment by leotsarev** [comment]
@Promaethius may be you have this trans-piled version somewhere? Or even container?

**Comment by jehy** [comment]
I managed to build a working version and put it into docker container, you can [check it here](https://github.com/jehy/cmx.js/blob/master/readme.md).

UPD. Later, I also removed all unused deps,, and now you can simply `npm ci` and `npm start` with node 22...

**Comment by prabuvenkat-gh** [comment]
@jehy Greatly appreciate you brushing this up. 

I am interested in using, extending and maintaining this. Will be nice if @darwin or you can give a headstart with some simple/basic documentation/info (ref #6)!

New to coffeescript, but am willing to learn!..

**Comment by jehy** [comment]
> @jehy Greatly appreciate you brushing this up.
> 
> I am interested in using, extending and maintaining this. Will be nice if @darwin or you can give a headstart with some simple/basic documentation/info (ref #6)!
> 
> New to coffeescript, but am willing to learn!..

Hi! You can contact me via telegram, I can provide some details. We can also make a zoom call.
BTW, you should not learn coffescript. It's dead already. It's better to start with migrating to typescript. May be there are some automated migrators, or you can try chatgpt.

**Comment by prabuvenkat-gh** [comment]
Yes, wanted to suggest it myself, but did not think it was right to do so before we could have a discussion. I am a bit busy through this weekend, but we take it up next week!
